tests/component/test_task_events.py

```diff
@@ -55,6 +56,118 @@ def test___done_event_registered___run_finite_acquisition___callback_invoked_onc
     assert all(e.status == 0 for e in event_observer.events)
 
 
+@pytest.mark.grpc_xfail(
+    reason="#559: Can't close tasks in an event callback when using gRPC", raises=AssertionError
+)
+def test___done_event_registered___call_clear_in_callback___stop_close_in_callback_with_success_status(
```

Comment:
"call_clear_in_callback" describes the underlying C API call. The "act" section of this test does "call_stop_close_in_callback"

"stop_close_in_callback_with_success_status" sort of describes the result but I think "task_closed_with_success_status" matches the code better.

Reply:
Handled in #564